client/keys/add_ledger_test.go

```diff
@@ -79,14 +72,14 @@ func Test_runAddCmdLedgerWithCustomCoinType(t *testing.T) {
 	pub, err := key1.GetPubKey()
 	require.NoError(t, err)
 	require.Equal(t,
-		"PubKeySecp256k1{03028F0D5A9FD41600191CDEFDEA05E77A68DFBCE286241C0190805B9346667D07}",
+		"PubKeySecp256k1{03028F0D5A9FD41600191CDEFDEA05E77A68DFBCE286241C0190805B9346667D070000000000000000000000000000000000000000000000000000000000000000}",
```

Comment:
We have a compressed public key with length 65 bytes?

Reply:
Yeah as discussed in DMs,I suspect we use the modified `crypto` package with uncompressed key, and  the ledger `keyring` material wasn't changed accordingly. Since we created tasks for ledger tests, I believe we can resolve this.